packages/docs/blocks/display/S3Download.yaml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020-2024 Lowdefy, Inc
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_ref:
+  path: templates/blocks/template.yaml.njk
+  vars:
+    block_type: S3Download
+    value_type: object
+    category: display
+    schema: ../plugins/plugins/plugin-aws/src/blocks/S3Download/schema.json
+    filePath: blocks/display/S3Download.yaml
+    description_content: |
+      The `S3Download` block allows a user to download a files from AWS S3.
+
+      The `s3GetPolicyRequestId` property is required, and should be the `id` of a [`AwsS3PresignedGetPolicy`](/AWSS3) request. This request is called by the block when the user downloads a file, to create a policy that allows a file to be downloaded from AWS S3.
+
+      If using files was uploaded using a S3Upload block, the `fileList` can be mapped to the `S3Download` block to allow the user to download the files, given that the correct `AwsS3PresignedGetPolicy` is in defined.
+
+      For the examples to work, a `AwsS3Bucket` connection is required. For example:
+      ```yaml
+        connections:
+        - id: files
+          type: AwsS3Bucket
+          properties:
+            accessKeyId:
+              _secret: FILES_S3_ACCESS_KEY_ID
+            secretAccessKey:
+              _secret: FILES_S3_SECRET_ACCESS_KEY
+            region: af-south-1
+            bucket:
+              _secret: FILES_S3_BUCKET
+            write: true
+      ````
+
+      > The examples on this page is not setup with a [`AwsS3Bucket`](/AWSS3) connection, and thus will throw.
```

Comment:
The examples on this page are not set up with an [`AwsS3Bucket`](/AWSS3) connection